pkgs/development/python-modules/f90nml/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, pythonPackages, setuptools-scm }:
+
+with pythonPackages;
+
+buildPythonPackage rec {
+  pname = "f90nml";
+  version = "1.4.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-nfMSqhO5whk28FnKuatAr+vCgPGrVOaFTDhz0Le3hlw=";
+  };
+
+  nativeBuildInputs = [ setuptools-scm ];
+
+  doCheck = false;
```

Comment:
There are tests https://github.com/marshallward/f90nml we want to execute and a pythonImportsCheck is missing.

Reply:
Just added the `pythonImportsCheck` but couldn't make the tests to works. Tests are looking for test files which they cannot find.

Reply:
You probably need to fetch from github for that to work.

Reply:
Indeed the test files are not distributed on Pypi.